hw2/05-form.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>05 - Form</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
+    <script defer src="05-form.js"></script>
+  </head>
+  <body class="bg-dark">
+    <main>
+      <form class="bg-light border rounded w-50 mx-auto mt-5 p-3">
+        <h3 class="w-50 mx-auto mt-2 mb-4">Exercise 05 - Form</h3>
```

Comment:
You might consider making this an h1 heading so it is more accessible.

Reply:
I was trying to match the screenshot and an h1 seemed too big. Accessibility is more important though, I'll make the change.